src/manifest/dbtProject.ts

```diff
@@ -785,25 +429,13 @@ export class DBTProject implements Disposable {
   }
 
   async generateSchemaYML(modelPath: Uri, modelName: string) {
```

Comment:
these error handlers are not needed anymore?

Reply:
Will be handled as follow up (not sure if they are needed or not)